src/main/views/live-case-status.njk

```diff
@@ -14,9 +14,10 @@
 {% endblock %}
 
 {% block content %}
-    <h2 class="govuk-heading-l">Live hearing updates</h2>
+    <h2 class="govuk-heading-l">{{ heading }}</h2>
     <h3 class="govuk-heading-m">{{ courtName }}</h3>
-    <p class=" govuk-body-lead">{{ updateDateTime }}</p>
+    <p class=" govuk-body-lead">{{ updateDate }}</p>
+    <p class=" govuk-body">{{ lastUpdatedText }}{{ updateTime }}</p>
     <p class="govuk-body">The status of each hearing will be updated as events happen live in court</p>
```

Comment:
Can you please use the locales value for this one as well

Reply:
Hi Maro - should be done in just now's commit :)